Makefile

```diff
@@ -2,6 +2,7 @@ all: generate
 
 .PHONY: generate
 generate:
+	cp ../docs/static/specs/services/coordinator/public/v1/public_api.swagger.json inputs/public_api.swagger.json
```

Comment:
this seems dangerous because there's quite a few times where I want to tweak my local input and regenerate without pulling the spec from mono.

Let's make a separate make target for this? `make sync`?